packages/compass-aggregations/src/modules/focus-mode.ts

```diff
@@ -40,13 +40,13 @@ export default function reducer(state = INITIAL_STATE, action: AnyAction): State
   return state;
 }
 
-export const focusModeEnabled = (
+export const enableFocusMode = (
```

Comment:
`focusModeEnabled` reads like boolean flag, not an action, so I renamed it to avoid confusion